srcpkgs/arc-theme/template

```diff
@@ -0,0 +1,16 @@
+pkgname=arc-theme
+version=20170302
+revision=1
+build_style=gnu-configure
+noarch=yes
+makedepends="pkg-config automake autoconf gtk+3-devel"
+short_desc="Arc is a flat theme with transparent elements for GTK 3, GTK 2 and GNOME Shell"
```

Comment:
`short_desc="Flat theme with transparent elements for GTK 3, GTK 2 and GNOME Shell"`